src/PHPMailer.php

```diff
@@ -158,11 +158,20 @@ class PHPMailer
     public $Ical = '';
 
     /**
-     * Value of "method" in Contenttype header "text/calendar"
+     * Value-array of "method" in Contenttype header "text/calendar"
      *
-     * @var string
+     * @var string[]
      */
-    public $IcalMethod = self::ICAL_METHOD_REQUEST;
+    private static $IcalMethod = [
```

Comment:
I was wondering why you hadn't used a constant array here - then I checked and found that you couldn't do that prior to PHP 5.6, and we are meant to retain compatibility back to PHP 5.5 in the 6.x branch, so that will have to wait. That said, this should be `protected` rather rather than `private` so that users are free to override it to add or remove Ical methods. I'd also call it `IcalMethods` since it's an array, not a single value.

Reply:
Yes, the constant thing will not work by now. I also changed it to case-insensitive. I worked with many groupwaresystems, i think it was Zarafa or Groupwise, in some of these ich came up with all lowercase icalendars. I will change it to protected, good idea. The thing with IcalMethods came to me too, just after commiting, sorry for that one.